team-13.md

```diff
@@ -0,0 +1,148 @@
+# Chrome Extension Idea: TypeMaster (tentative)
+
+## Authors
+
+Tiffany Lian, Alexander Dempsey, James Huang, Sanya Shetty
+
+## Problem Statement
```

Comment:
Lovely proposal. You did a good job communicating exactly how the word counter + tone analysis extension would benefit multiple users. The users story did a great job at outlining how that various features features would functions. 

A note on the problem statement:
I would refrain from mentioning the solution in the problem statement, so maybe introduce the integration of real-time AI-powered semantic analysis in the product description.